package.json

```diff
@@ -32,7 +34,8 @@
     "stellar-hd-wallet": "^0.0.10",
     "stellar-sdk": "5.0.1",
     "styled-components": "^5.0.1",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "yup": "^0.29.1"
```

Comment:
adding yup, a schema forming library.

formik validation is very open ended, and not using Yup makes using Formik almost not worth it